Source/Fuse.Maps/MapTile.uno

```diff
@@ -0,0 +1,298 @@
+using Fuse.Elements;
+using Uno.UX;
+using Uno;
+using Fuse.Maps;
+using Uno.Collections;
+
+namespace Fuse.Controls
+{
+	/** A polyfill for Native @MapView
+
+	This is not meant to be a full implementation, but just something that will help you when developing on local preview.
+
+	This should also work as a standalone component, so it can be used on native.block
+
+	This uses the OSM tile server, and that has both a usage policy https://operations.osmfoundation.org/policies/tiles/ and a license https://www.openstreetmap.org/copyright .
+
+	The mappin is from wikipedia https://commons.wikimedia.org/wiki/File:Map_pin_icon.svg and is Creative Commons Attribution-Share Alike 3.0 Unported. 
+
+	@seealso Fuse.Controls.MapView
+	*/
+
+	public class MapTile : Panel
```

Comment:
Without a solid use-case it'd be better to keep it internal. It provides a valuable service for MapView, but on it's own it's uncertain. 

If it's public then we'd have to go through an API review, require examples, more docuemtnation, and also be supported in future version. All things we'd like to avoid where possible.